pref/src/main/kotlin/voice/pref/AutoSleepTimerPrefs.kt

```diff
@@ -0,0 +1,10 @@
+package voice.pref
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class AutoSleepTimerPrefs(
+  var enabled: Boolean,
```

Comment:
Generally its a good practice to keep as much immutable as possible.
For example we collect this from a flow.
That means if you would now call

prefs.enabled = false

That change would never properly propagate.
I recommend to make these three properties `val`s to prevent accidental misusage